models/data_warehouse.model.lkml

```diff
@@ -3995,6 +3992,18 @@ explore: user_retention {
   label: " User Retention"
   group_label: " Product: Messaging"
   hidden: no
+  fields: [ALL_FIELDS*, -user_events_telemetry.stripe_customer_email, -user_events_telemetry.stripe_customer_dns
```

Comment:
What are these for @sri-byte?

Reply:
Looker is throwing an error, because of joins in the events table which don't apply in user_retention. I had to read through some documentation and apply this fix so the error goes away. 
![image](https://user-images.githubusercontent.com/96433562/166809433-9adca963-2826-424b-b0e6-f2d93bcd1cae.png)

Reply:
Thanks @sri-byte, I'll go ahead and approve the changes